settings.gradle.kts

```diff
@@ -16,6 +16,8 @@ rootProject.name = "Kotlin-DateTime-library"
 
 include(":core")
 project(":core").name = "kotlinx-datetime"
+include(":timezones/full")
+project(":timezones/full").name = "kotlinx-datetime-timezones-full"
```

Comment:
The artifact name we decided on is `kotlinx-datetime-zoneinfo`. At least for now, without `full`.

Reply:
And if we decide to make an intermediate timezones db?

Reply:
When we decide to do it, we'll know from the reported use cases what exactly people want, and then we'll choose the names appropriately.

Reply:
`-zoneinfo`, not `-timezones`.